cloud/pkg/common/client/client.go

```diff
@@ -56,6 +59,15 @@ func InitKubeEdgeClient(config *cloudcoreConfig.KubeAPIConfig) {
 		crdKubeConfig := rest.CopyConfig(kubeConfig)
 		crdKubeConfig.ContentType = runtime.ContentTypeJSON
 		crdClient = crdClientset.NewForConfigOrDie(crdKubeConfig)
+
+		authKubeConfig, err = clientcmd.BuildConfigFromFlags(kubeConfig.Host, "")
+		if err != nil {
+			klog.Errorf("Failed to build config, err: %v", err)
+			os.Exit(1)
+		}
+		authKubeConfig.CAData = kubeConfig.CAData
+		authKubeConfig.CAFile = kubeConfig.CAFile
```

Comment:
Seems no need to set the certs here as we have token below?

Reply:
CA file or CA data still needed, it will be authenticated by kube-apiserver.